web/src/loading.ts

```diff
@@ -92,3 +92,27 @@ export function show_button_spinner($elt: JQuery, using_dark_theme: boolean): vo
     }
     $elt.css("display", "inline-block");
 }
+
+// For showing spinners at dialog_widget and user_profile.
+export function show_spinner($button: JQuery): void {
+    const $spinner = $button.find(".modal__spinner");
+    const dialog_submit_button_span_width = $button.find("span").width();
+    const dialog_submit_button_span_height = $button.find("span").height();
```

Comment:
Given, we rename the function as suggested above, let's rename these constants to `submit_button_span_width`/`submit_button_span_height` for now.